client/components/checkout-form.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { CSSTransitionGroup } from 'react-transition-group';
+
+export default class CheckoutForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      card: '',
+      address: ''
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleReset = this.handleReset.bind(this);
+  }
+
+  handleChange(event) {
+    const property = event.target.name;
+    this.setState({
+      [property]: event.target.value
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    this.setState({
+      name: '',
+      card: '',
+      address: ''
+    });
+  }
+
+  handleReset() {
+    this.setState({
+      name: '',
+      card: '',
+      address: ''
+    });
+    this.props.setView('catalog', {});
+  }
+
+  render() {
+    const submitInfo = { name: this.state.name, creditCard: this.state.card, shippingAddress: this.state.address };
+    return (
+      <CSSTransitionGroup
+        transitionName="example"
+        transitionAppear={true}
+        transitionAppearTimeout={500}
+        transitionEnter={false}
+        transitionLeave={false}>
```

Comment:
I wonder if it would be cleaner to move this transition element into App's render method since all of your screens transition?